### DISCUSSION TITLE
Colored Glass

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1203] [Help] [answered]
Hi, how can I create colored glass? 
In bukkit you can do:
```
new ItemStack(Material.GLASS, 1, (short) 4);
```
~BlokJochem

**Comment by MrlnHi** [comment, marked as answer]
In Minestom (and also in Bukkit post 1.8) you do
`Block.LIGHT_BLUE_GLASS` and so on

**Reply by DevBlokJochem** [reply]
Thanks. I havent worked that mutch with the new 1.13+ api.
